frame/system/src/mock.rs

```diff
@@ -34,12 +34,11 @@ impl_outer_origin! {
 #[derive(Clone, Eq, PartialEq, Debug, Default)]
 pub struct Test;
 
+const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
+const TARGET_BLOCK_WEIGHT: Weight = 1024;
```

Comment:
We use the word `TARGET_BLOCK_WEIGHT ` in the context of multiplier update with a special meaning, so I recommend calling this just 
```suggestion
const MAX_BLOCK_WEIGHT: Weight = 1024;
```

To avoid confusion.